node/sc/protocol.go

```diff
@@ -38,10 +38,11 @@ const (
 	ServiceChainReceiptRequestMsg          = 0x03
 	ServiceChainParentChainInfoResponseMsg = 0x04
 	ServiceChainParentChainInfoRequestMsg  = 0x05
+	ServiceChainInvalidTxResponseMsg       = 0x06
 
-	ServiceChainCall     = 0x06
-	ServiceChainResponse = 0x07
-	ServiceChainNotify   = 0x08
+	ServiceChainCall     = 0x07
+	ServiceChainResponse = 0x08
+	ServiceChainNotify   = 0x09
```

Comment:
This kind of protocol update can make a mismatch in the message protocol.
Why don't you increase the protocol version? We can consider a better protocol version management policy also

Reply:
What a great catch! Thank you so much. The SC protocol version scope is currently going on another PR(#1248). Once that PR is finalized, let me update it.